sip-archetype/src/main/resources/archetype-resources/adapter-description.md

```diff
@@ -0,0 +1,5 @@
+# Welcome!
+
+**Thank you for using SIP Framework!**
+
+Please describe your adapter workflow.
```

Comment:
Just a bit more text, like: 
This file should contain details about what your adapter does, which services does it connect, what is the workflow etc.